File: Boucles/TP4_6.js
function sumArray(array) {
  let i = 0;
  let sum = 0;
  let currentElement;
  while (i < array.length) {
    currentElement = array[i];
    sum = sum + currentElement
    i++;
  }
  console.log(sum);
  return sum;
}

let test = [12, 13, 14];
sumArray(test);

File: Boucles/TP4_6.py
def sumArray(array) :
  i = 0;
  sum = 0;
  while (i < len(array)) :
    currentElement = array[i];
    sum = sum + currentElement
    i+=1;

  print(sum);
  return sum;


test = [12, 13, 14];
sumArray(test)
